openedx/core/djangoapps/content/course_overviews/models.py

```diff
@@ -97,6 +98,7 @@ class Meta(object):
     course_video_url = TextField(null=True)
     effort = TextField(null=True)
     self_paced = BooleanField(default=False)
+    instructor_info = JSONField(null=False, blank=False)
```

Comment:
Don't we need a dict as a default.
Sample data for this field is like

`{'instructors': [
{instructor1 data},
{instructor2 data}
]}`

or am I missing something ?